tox.ini

```diff
@@ -35,3 +34,43 @@ deps =
     pylint
 whitelist_externals =
     sh
+
+[testenv:black]
+skip_install = true
+deps = black
+commands = black --skip-string-normalization .
+
+[testenv:linters]
+skip_install = true
+basepython = python3
+deps = flake8
+       flake8-colors
+       black
+commands =
+  # Don't fail on black failure for now
+  -black --skip-string-normalization --check .
```

Comment:
Maybe you can add a commit with the code blackified and remove the '-'. Otherwise I'm good.

Reply:
That's a long standing one that is failing in the integration tests not really a linter issue...

Reply:
Please don't blackify the code before merging all branches (particularly `use-real-calld-client`, still need ready)